sourcegraph/about-replatform
# Add non-animated customer logos section and update contact pages

Closes #118 by adding our non-animated logos component and updating the contact pages where that component was previously commented out.

## To Test
- Check out the contact pages where this component gets used and ensure that everything looks good on desktop/mobile
- Check static builds
